src/adaptations/device-layer/EFR32/freertos_bluetooth.c

```diff
@@ -0,0 +1,399 @@
+/***************************************************************************/ /**
```

Comment:
This file needs an OpenWeave licence header.

Reply:
Done